v2.1/reference/tools/pd-control.md

```diff
@@ -22,7 +22,7 @@ As a command line tool of PD, PD Control obtains the state information of the cl
 Single-command mode:
 
 ```bash
-./pd-ctl store -d -u http://127.0.0.1:2379
+./pd-ctl store -u http://127.0.0.1:2379
```

Comment:
Why change this line? See https://github.com/pingcap/docs-cn/pull/1818#issuecomment-527870946